app/app.js

```diff
@@ -1,19 +1,18 @@
 'use strict';
 
 // Declare app level module which depends on views, and components
-angular
-  .module('app', [
-    'ngRoute',
-    'ngStorage',
-    'app.home',
-    'app.player',
-    'app.round'
-  ])
-
-  .config([
-    '$routeProvider',
-    function($routeProvider) {
-      // Redirects user to the main page if the selected one does not exist
-      $routeProvider.otherwise({redirectTo: '/'});
-    }
-  ]);
+angular.module('app', [
```

Comment:
These diffs show that you changed the indentation of the file itself. Please add only what u need without changing the indentation